scripts/travis_ci/travis_script.sh

```diff
@@ -28,7 +28,14 @@ if [ "$RUN_X86_TESTS" = "true" ]; then
     BUILD_DIR=boards/x86_build
     travis_run cmake -S boards -B $BUILD_DIR -DPLATFORM=x86
     travis_run make --directory=$BUILD_DIR
-    travis_run cd $BUILD_DIR && ctest && cd -
+    travis_run "cd $BUILD_DIR && ctest"
+
+    if [ $? -ne 0 ]; then
```

Comment:
Nit; can we capture the return value in a variable and check that instead? I think it makes the code more readable and less error-prone:
```
CTEST_RETURN_CODE=$?
if [ $CTEST_RETURN_CODE -ne 0 ]; then
...
```